### DISCUSSION TITLE
拖拽源码的问题

### DISCUSSION THREAD

**Discussion by twinkle77** [alibaba/designable#111] [Q&A] [answered]
https://github.com/alibaba/designable/blob/main/packages/core/src/drivers/DragDropDriver.ts#L89

![image](https://user-images.githubusercontent.com/17792166/137891088-b200d53b-acfd-4d14-b7c3-7f19a70607fe.png)

请问下，为什么即监听 dragover 又 监听 mousemove 事件？重复了？

**Comment by janryWang** [comment, marked as answer]
为了支持有draggble属性的元素

**Reply by twinkle77** [reply]
ok 谢谢
